app/post_order.py

```diff
@@ -0,0 +1,20 @@
+def postorder_traversal(root):
```

Comment:
**Code optimization Suggestion**
```python
def postorder_traversal(root):
    result = []
    def traverse(node):
        nonlocal result
        if node:
            traverse(node.left)
            traverse(node.right)
            result.append(node.value)
    traverse(root)
    return result
```
**Explanation for suggestion**
Refactored to use a single list and append to it during traversal, avoiding the creation of multiple lists and concatenation operations, which improves efficiency.